build.gradle

```diff
@@ -34,6 +34,7 @@ allprojects {
 		compileOnly "org.jetbrains:annotations:${project.jetbrains_annotations_version}"
 
 		testImplementation "org.junit.jupiter:junit-jupiter:${junit_jupiter_version}"
+		testImplementation "org.jooq:joor-java-8:${project.joor_version}"
```

Comment:
Do we really need a library for this? I know its test only but I am struggling to see the benefit in it.

Reply:
It does make reflection [nicer](https://github.com/jOOQ/jOOR#readme) to work with (shorter queries, no checked exceptions etc.), but for simple constructor invocations like here, it's not quite necessary, I agree. I would definitely add it back in if we needed to do method invocations in the future though, those become very tedious and convoluted very quickly.